src/AggregateSpecsPlugin.ts

```diff
@@ -48,10 +86,12 @@ const AggregateSpecsPlugin: Plugin = (builder) => {
         id: "sum",
         humanLabel: "sum",
         HumanLabel: "Sum",
-        isSuitableType: isNumberLike,
+        isSuitableType: (pgType) => isNumberLike(pgType) || isInterval(pgType),
         // I've wrapped it in `coalesce` so that it cannot be null
-        sqlAggregateWrap: (sqlFrag) =>
-          sql.fragment`coalesce(sum(${sqlFrag}), 0)`,
+        sqlAggregateWrap: (sqlFrag, pgType) =>
+          sql.fragment`coalesce(sum(${sqlFrag}), ${
+            SUM_DEFAULTS[pgType.id] || "0"
```

Comment:
I would not expect this to work - it should throw an error due to "expected SQL item" or similar. ~~You need to wrap it in `sql.value` (or, in this case, better to use `sql.literal` so it gets inlined without using a placeholder if possible):~~
```
            sql.literal(SUM_DEFAULTS[pgType.id] || "0")
```

Correction; the `SUM_DEFAULTS` seem to need to be SQL fragments due to casting, so this should be an SQL fragment fallback:

```suggestion
            SUM_DEFAULTS[pgType.id] || sql.fragment`0`
```